### DISCUSSION TITLE
Anyone having problems with DNN ImageHandler and DNN 9.8.0?

### DISCUSSION THREAD

**Discussion by roman-yagodin** [dnnsoftware/Dnn.Platform#4352] [General]
Having clean DNN 9.8.0 install, and the only thing that worked is `profilepic` mode. Other modes including `securefile`, `file`, `placeholder` produce *403 - Forbidden: Access is denied* HTTP errors, but no errors in log. Do I miss something?

Already checked all available docs including https://dnndocs.com/content/tutorials/developers/image-handler/index.html#image-handler but still have no clue.

**Comment by bdukes** [comment]
It does definitely look like this is a widespread issue.  Can you open an issue?

**Comment by roman-yagodin** [comment]
Sure! #4355

**Comment by sleupold** [comment]
Thank you, @roman-yagodin !
